projects/cloud/app/graphql/mutations/change_user_role.rb

```diff
@@ -0,0 +1,14 @@
+module Mutations
+  class ChangeUserRole < ::Mutations::BaseMutation
+    argument :user_id, ID, required: true
+    argument :organization_id, String, required: true
+    argument :current_role, Types::RoleType, required: true
```

Comment:
Because the `current_role` data is already in `user` (fetched from `user_id`), I don't think we need to pass this from the client.